tests/components/paperless_ngx/conftest.py

```diff
@@ -0,0 +1,63 @@
+"""Common fixtures for the Paperless-ngx tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from pypaperless.models import Statistic
+import pytest
+
+from homeassistant.components.paperless_ngx.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from . import setup_integration
+from .const import MOCK_STATISTICS_DATA, USER_INPUT
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(autouse=True)
+def mock_paperless() -> Generator[AsyncMock]:
+    """Mock the pypaperless.Paperless client."""
+    with (
+        patch(
+            "homeassistant.components.paperless_ngx.coordinator.Paperless",
+            autospec=True,
+        ) as paperless_mock,
+        patch(
+            "homeassistant.components.paperless_ngx.config_flow.Paperless",
+            new=paperless_mock,
+        ),
+    ):
+        paperless = paperless_mock.return_value
+
+        paperless.base_url = "http://paperless.example.com/"
+        paperless.host_version = "2.3.0"
+        paperless.initialize = AsyncMock(return_value=None)
+        paperless.statistics = AsyncMock(
+            return_value=Statistic.create_with_data(
+                paperless, data=MOCK_STATISTICS_DATA, fetched=True
+            )
+        )
+
+        yield paperless
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        entry_id="paperless_ngx_test",
```

Comment:
Personally I would prefer a generated string (that generally an entry_id is) over this, as then it's always clear that its an entry_id